### ISSUE TITLE
Выдает неправильную ошибку при использовании функции exit

### ISSUE THREAD

**Issue by MrFresnel** [pascalabcnet/pascalabcnet#1351]
```
function F() := exit();

begin
end.
```
Ошибка:
> Program2.pas(1) : Ожидалась функция, встречена процедура none

На мой взгляд, правильно было бы написать следующий текст:
> Возвращаемое значение функции exit не может быть использована в качестве возвращаемого значения функции или метода

**Comment by miks1965** [comment]
exit - не функция - с чего Вы взяли? Это специальная внутренняя процедура. Кстати, во Free Pascal - тоже процедура. Ровно такая же как у нас

**Comment by SunSerega** [comment]
Нет, ошибка всё равно неправильная. Там `none` надо заменить на `exit`.

**Comment by miks1965** [comment]
Подумаем. На том уровне, на котором это выводится, она давно уже не exit, а сильно внутренняя безымянная процедура
